packages/scene-graph-mediator/runtime/src/importer/Importer.ts

```diff
@@ -211,4 +216,15 @@ export abstract class Importer {
 
     return objectMap;
   }
+
+  protected createRuntimeObjectForPlugins(node: Node, resources: any): any | null {
```

Comment:
To clarify that `createRuntimeObjectForPlugins` may skip  `createRuntimeObject`.
I think `createRuntimeObjectForPlugins` may be executed in `createRuntimeObject`.
How do you think about this atomicity ?

Reply:
In that case, whether to consider a plug-in depends on the class that inherits the importer.

However, the scene-graph-mediator implementation may be more appropriate because it is customized on a large scale. I give priority to that.

And if you do not process upstream, you need to do the same for the "Three" importer.

Reply:
@dolow 
How about this implementation?
7100faeeab0bbfefff70280bc174c325a9934b9c

Reply:
@takeshi-sakurai 
Seems fine, added comment about little thing.